tasks/install.yml

```diff
@@ -38,6 +38,7 @@
   delay: 2
   # run_once: true
   delegate_to: localhost
+  check_mode: no
```

Comment:
Needs to be `false` otherwise linter complains.

Reply:
No problem, I've made that change.